src/skills-builder/skills-builder-modal/select-preferences/CareerInterestSelect.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const CareerInterestSelect = () => (
+  <div>
+    <h4>What careers are you interested in?</h4>
```

Comment:
Should this be hard-coded English?

Reply:
This was just placeholder for the next ticket, APER-2249, but yes it will need to be translated so I'll set that up here so it's accounted for